src/pages/index.vue

```diff
@@ -23,16 +23,17 @@
       <VSearchBar
         v-model.trim="searchTerm"
         class="max-w-[40rem] mt-4 lg:mt-8"
-        :placeholder="featuredSearchText"
         @submit="handleSearch"
       >
-        <VContentSwitcherPopover
-          v-if="isMounted && isMinScreenMd"
-          ref="contentSwitcher"
-          class="mx-3"
-          :active-item="contentType"
-          @select="setContentType"
-        />
+        <ClientOnly>
```

Comment:
It might be hard to understand why this is client only (because I don't get it). Do you think an explanatory comment `<!-- -->` here might be justified?

Reply:
I don't know why it is client only, I was actually hoping you did! `ClientOnly` just removes the extra logic that we previously had for the `isMounted` ref that was being maintained.

Reply:
🤷